deep-learning/_posts/2018-06-29-simpleDoesIt.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: review
+title:  "Simple Does It: Weakly Supervised Instance and Semantic Segmentation"
+tags:   deep-learning CNN segmentation weakly-supervised
+author: "Pierre-Marc Jodoin"
+pdf: "http://openaccess.thecvf.com/content_cvpr_2017/papers/Khoreva_Simple_Does_It_CVPR_2017_paper.pdf"
+cite:
+  authors: "Anna Khoreva, Rodrigo Benenson, Jan Hosang, Matthias Hein, B Schiele" 
+  title:   "Simple Does It: Weakly Supervised Instance and Semantic Segmentation"
+  venue:   "CVPR 2018"
```

Comment:
CVPR 2017 no? See the url

Also the code :

Reply:
crap! let me correct this.